java-checks/src/test/files/checks/ClassWithOnlyStaticMethodsInstantiationCheck.java

```diff
@@ -0,0 +1,59 @@
+class A {
```

Comment:
Inner classes can not be static. See [jls 8.1.3](http://docs.oracle.com/javase/specs/jls/se8/html/jls-8.html#jls-8.1.3).